app/components/node/PoSFooter.tsx

```diff
@@ -12,19 +12,31 @@ const Footer = styled.div`
   margin-top: 25px;
 `;
 
+const ButtonWrap = styled.div`
+  display: flex;
+  width: 250px;
+  justify-content: flex-end;
+  flex-direction: row;
+`;
+
 type Props = {
   action: () => void;
   isDisabled: boolean;
   isLastMode?: boolean;
+  isFirstMode?: boolean;
+  skipAction: () => void;
 };
 
 class PoSFooter extends PureComponent<Props> {
   render() {
-    const { action, isDisabled, isLastMode } = this.props;
+    const { action, isDisabled, isLastMode, isFirstMode, skipAction } = this.props;
```

Comment:
'skipAction' is enough to decide if to render or not
also make it optional with default value of 'undefined'

Reply:
you are right